code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,259 @@
+	QDEL_NULL(current_bandage)
+	current_bandage = new I.type(limb)
+	current_bandage.amount = 1
+	treat_priority = FALSE
+	I.use(1)
+
+
+/datum/wound/brute/cut/moderate
+	name = "Rough Abrasion"
+	desc = "Patient's skin has been badly scraped, generating moderate blood loss."
+	treat_text = "Application of clean bandages or first-aid grade sutures, followed by food and rest."
+	examine_desc = "has an open cut"
+	occur_text = "is cut open, slowly leaking blood"
+	severity = WOUND_SEVERITY_MODERATE
+	initial_flow = 2.5
+	minimum_flow = 0.5
+	max_per_type = 3
+	clot_rate = 0.15
+	threshold_minimum = 20
+	threshold_penalty = 10
+	status_effect_type = /datum/status_effect/wound/cut/moderate
+	scarring_descriptions = list("light, faded lines", "minor cut marks", "a small faded slit", "a series of small scars")
+
+/datum/wound/brute/cut/severe
+	name = "Open Laceration"
+	desc = "Patient's skin is ripped clean open, allowing significant blood loss."
+	treat_text = "Speedy application of first-aid grade sutures and clean bandages, followed by vitals monitoring to ensure recovery."
+	examine_desc = "has a severe cut"
+	occur_text = "is ripped open, veins spurting blood"
+	severity = WOUND_SEVERITY_SEVERE
+	initial_flow = 3.75
+	minimum_flow = 3
+	clot_rate = 0.05
+	max_per_type = 4
+	threshold_minimum = 50
+	threshold_penalty = 25
+	demotes_to = /datum/wound/brute/cut/moderate
+	status_effect_type = /datum/status_effect/wound/cut/severe
+	scarring_descriptions = list("a twisted line of faded gashes", "a gnarled sickle-shaped slice scar", "a long-faded puncture wound")
+
+/datum/wound/brute/cut/critical
+	name = "Weeping Avulsion"
+	desc = "Patient's skin is completely torn open, along with significant loss of tissue. Extreme blood loss will lead to quick death without intervention."
+	treat_text = "Immediate surgical suturing and bandaging followed by supervised resanguination."
+	examine_desc = "is spurting blood at an alarming rate"
+	occur_text = "is torn open, spraying blood wildly"
+	severity = WOUND_SEVERITY_CRITICAL
+	initial_flow = 5
+	minimum_flow = 4.5
+	clot_rate = -0.05 // critical cuts actively get worse instead of better
+	max_per_type = 5
+	threshold_minimum = 80
+	threshold_penalty = 40
+	demotes_to = /datum/wound/brute/cut/severe
+	status_effect_type = /datum/status_effect/wound/cut/critical
+	scarring_descriptions = list("a winding path of very badly healed scar tissue", "a series of peaks and valleys along a gruesome line of cut scar tissue", "a grotesque snake of indentations and stitching scars")
+
+// TODO: see about moving dismemberment over to this, i'll have to add judging dismembering power/wound potential wrt item size i guess
+/datum/wound/brute/cut/loss
+	name = "Dismembered"
+	desc = "oof ouch!!"
+	occur_text = "is violently dismembered!"
+	viable_zones = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
```

Comment:
what about the head?

Reply:
Head dismemberment is much more sticky balance wise than limbs. This is just a proof of concept really anyway